common/lib/dependabot/pull_request_creator/message_builder/metadata_presenter.rb

```diff
@@ -244,7 +244,7 @@ def source_provider_supports_html?
         end
 
         def sanitize_links_and_mentions(text, unsafe: false)
-          return text unless source.provider == "github"
+          return text unless source.provider == "github" || source.provider == "gitlab"
```

Comment:
If I understand correctly we need something like...

```ruby
return text if running_on?(:github) && source.provider == 'github'
return text if running_on?(:gitlab) && source.provider == 'gitlab'
```

Does that capture the reason that this PR is blocked?

Reply:
From what I understand, the issue is determining the source of `Release notes` and `Changelog` so we do not sanitize links pointing natively to Gitlab as Github links (I have not encountered such a case myself though).

If I understand correctly, `source.provider` is not that, it is basically a pointer where the dependabot-core runs and not where changelog comes from.